Beginner_Projects/Tune_Tracker/Tune_tracker.py

```
@@ -0,0 +1,169 @@
+import random
+
+def add_song(songs):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)